src/ima-state-machine.js

```diff
@@ -555,19 +561,16 @@ function getAdBreakOptions(adEvent: any): Object {
  */
 function getAdBreakType(adEvent: any): string {
   const ad = adEvent.getAd();
-  const podInfo = ad.getAdPodInfo();
```

Comment:
is there a more "less breaking" way to do this? still using IMA API to get type when possible?

Reply:
done